src/plugins/date/date.c

```diff
@@ -0,0 +1,347 @@
+/**
+ * @file
+ *
+ * @brief Source for date plugin
+ *
+ * @copyright BSD License (see doc/LICENSE.md or http://www.libelektra.org)
+ *
+ */
+
+#define _XOPEN_SOURCE
+#include "date.h"
+#include <ctype.h>
+#include <kdberrors.h>
+#include <kdbhelper.h>
+#include <langinfo.h>
+#include <locale.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+
+static int stricmp (const char * s1, const char * s2)
```

Comment:
i'd like to say the reason for not using strcasecmp is that it's nonstandard and not portable... but the actual reason: i had no idea strcasecmp actually existed.